seed/challenges/01-front-end-development-certification/advanced-bonfires.json

```diff
@@ -324,9 +324,15 @@
       "id": "a2f1d72d9b908d0bd72bb9f6",
       "title": "Make a Person",
```

Comment:
@justinian336 suggested a good translation (from it I just translated first line and escaped \<. We will wait few days that @eeseesee updates the pull request adding the translation (by changing in your branch and running <code>git commt -a --amend</code>).